### ISSUE TITLE
Synthetic data does not match real data in PAR model

### ISSUE THREAD

**Issue by yamidibarra** [sdv-dev/SDV#951]
### Environment details

SDV version: 0.16.0
Python version: 3.8.13
Operating System: Windows 10

### Problem description

I´m trying to generate synthetic sensor data i.e Pressure curves from 0 to 40 seconds.. I had an issue with the sequence_index column that I could resolve by coverting float-type values (seconds) into datetime-type. After this I could sample new data but it doesn´t look similar to the real one. 

### What I would like to know

Is there any information about how the PAR model generates synthetic data?. Could you provide a paper to have a better idea about the method.?

### The code
```
data = pd.read_pickle('df_PAR.pkl')
data['Time'] = data['Time'].multiply(1E9)
data['Time'] = pd.to_datetime(data['Time'])

context_columns = ['POM', 'Mold Temperature [°C]', 'Injection velocity [cmm/s]', 'Holding pressure [bar]'] 
entity_columns = ['id']
sequence_index = 'Time'
model = PAR(entity_columns=entity_columns,  context_columns=context_columns,  sequence_index=sequence_index)
 
model.fit(data)
new_data = model.sample(1)
   
 # get seconds
new_data['Time']=new_data['Time'].apply(lambda x:'%02d.%06d' %(x.second, x.microsecond)).astype(float)
```

**Comment by yamidibarra** [comment]
I couldn´t upload the plot...

**Comment by npatki** [comment]
Hi @yamidibarra, you can find more information about the PAR model here: https://arxiv.org/pdf/2207.14406v1.pdf

It will be really helpful if you are able to share a graph or more information about how you are evaluating the synthetic data. (GitHub allows you to attach files by clicking at the bottom of the text box.)

In particular, I'm curious which columns you are using how you're comparing the single synthetic sequence against the multiple real sequences.

**Comment by yamidibarra** [comment]
thank you dear @npatki. This method is for sequential timeseries. For the case of sensor data, I don´t want to extent or extrapolate the series but to know how the curve looks like when changing an operation setting that of course will produce a change in the data being measured at the sensor, so I would expect a change in the y axis but not an extension/extrapolation in the x axis (time)

**Comment by npatki** [comment]
Hi @yamidibarra, PAR model is currently designed to learn the general distribution of the time axis and its correlations with other columns. As a consequence: 

1. It does not currently learn any fixed intervals or the original values of the time axis. Related: #829
2. It will extrapolate upon the values as the sequence length increases. You can currently control the length using the [`sequence_length`](https://sdv.dev/SDV/api_reference/timeseries/api/sdv.timeseries.deepecho.PAR.sample.html) parameter, but this may not guarantee a specific end time. Related: #684.

Please feel free to follow along the referenced issues and add your use case if it seems relevant.

**Comment by npatki** [comment]
Hi @yamidibarra, I'm closing this off in favor of continuing the discussion in the linked issues. Feel free to reply if there is more to discuss and I will reopen this issue.
